src/wrapper.rs

```diff
@@ -7,6 +7,7 @@ use std::ffi::c_void;
 use winapi::shared::ntdef::{LPCWSTR, LPWSTR};
 use winapi::shared::winerror::HRESULT;
 
+#[macro_export]
```

Comment:
Can we make sure this macro doesn't become public to the crate (I'm guessing it is now, but I'm not sure).